apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/(marketplace)/components/list-button.tsx

```diff
@@ -40,14 +46,35 @@ export const CreateListingButton: React.FC<CreateListingButtonProps> = ({
         </SheetTrigger>
         <SheetContent className="w-full overflow-y-auto sm:min-w-[540px] lg:min-w-[700px]">
           <SheetHeader>
-            <SheetTitle className="mb-5 text-left">{createText}</SheetTitle>
+            <SheetTitle className="text-left">{createText}</SheetTitle>
           </SheetHeader>
-          <CreateListingsForm
-            contract={contract}
-            type={type}
-            actionText={createText}
-            setOpen={setOpen}
+          <TabButtons
```

Comment:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yNOf1svJ8o3zjO7zQouZ/8defd67d-8d55-4df8-96c9-4c261ff8d375.png)

Let's change the tab names to something more descriptive. 

"Select NFT" instead of "Selection" and remove the "Select NFT" title in tab content 

Add a description in "Manual" tab: 
"Enter the details of NFT you want to list for sale manually"

Reply:
updated